fly.staging.toml

```
@@ -0,0 +1,18 @@
+# fly.toml app configuration file generated for rails-ic-staging on 2023-05-30T13:04:02+07:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "rails-ic-staging"
+primary_region = "sin"
+console_command = "/rails/bin/rails console"
+
+[http_service]
+  internal_port = 3000
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+  min_machines_running = 0
+  
+[env]
+  DEPLOYED_HOST = "rails-ic-staging.fly.dev"
```

Do we need to config other `env` variables here?

```suggestion
  DEPLOYED_HOST = "rails-ic-staging.fly.dev"
    RAILS_ENV = "production"
  MAILER_DEFAULT_HOST = "localhost"
  MAILER_DEFAULT_PORT = "3000"
  MAILER_SENDER = "Test <noreply@nimblehq.co>"
  AVAILABLE_LOCALES = "en"
  DEFAULT_LOCALE = "en"
  FALLBACK_LOCALES = "en"
```

---

We don't need this for `development` environment because they are already supplied in `config/application.yml`
```
default: &default
  DB_NAME: "google_search_ruby"
  DB_HOST: "localhost"
  DB_PORT: "5432"
  DB_USERNAME: "postgres"
  DB_PASSWORD: "postgres"
  MAILER_DEFAULT_HOST: "localhost"
  MAILER_DEFAULT_PORT: "3000"
  MAILER_SENDER: "Test <noreply@nimblehq.co>"
  AVAILABLE_LOCALES: "en"
  DEFAULT_LOCALE: "en"
  FALLBACK_LOCALES: "en"

development:
  <<: *default
```